pandas/tests/series/indexing/test_indexing.py

```diff
@@ -654,6 +654,37 @@ def test_timedelta_assignment():
     tm.assert_series_equal(s, expected)
 
 
+class TestNaTAssignment:
```

Comment:
do we need a class here?

Reply:
This is broken off from a PR that implemented several related tests.